src/menhir-recover/Makefile

```diff
@@ -0,0 +1,15 @@
+.PHONY: all clean
```

Comment:
Was this (menhir-recover) copied form another project somewhere?

Reply:
Yes, it is a slightly improved version of https://github.com/ocaml/merlin/tree/master/src/ocaml/preprocess/recover (the same algorithm I wrote for Merlin/OCaml parser).

Reply:
Would you upstream your changes into merlin?

Reply:
I still have some work to do to generalize the algorithm a bit, but I plan to upstream it either into a separate repository or into Menhir directly.